### DISCUSSION TITLE
dejalu(mail client) in os.js?

### DISCUSSION THREAD

**Discussion by ghost** [os-js/OS.js#825] [Q&A]
Hi.

#### Idea
- In the previous post [I asked if there would be a need for vscode in os.js](https://github.com/os-js/OS.js/discussions/819). I was happy with the feedback I received... 
- So... In this post, I would like to know if the community is interested in [dejalu(email client)](https://github.com/dinhvh/dejalu). I'm thinking of developing support for a set of apps that I really like within os.js 

#### concept
<img src="https://user-images.githubusercontent.com/111701513/207094445-773f33dd-1d09-456c-b6f3-25a9a518591b.png" width="400px"/>

#### why?
1. I have 6 applications that I want to send a pull request to os.js: vscode(1), nullboard(2), keepass(3), reveal.js(4), [tldraw](https://www.tldraw.com/)(5) and dejalu(6)
2. dejalu is email client and has great ui/ux... os.js doesn't have a mail client
3. I want to contribute to os.js with apps that have a great ui/ux
4. Before coding, it is necessary to evaluate whether the idea is good or bad

**Comment by ajmeese7** [comment]
Feel free to add whatever you want, if the app is good enough I'm sure Anders will be open to creating an official OS.js repository for it. Applications are usually their own repository, so I do not recommend making a pull request on any existing repositories for a new app, as it will be denied.

**Reply by ghost** [reply]
Hi ajmeese7, thank you for feedback.
